docs/0.6/howto/removing_a_splinter_circuit.md

```diff
@@ -0,0 +1,339 @@
+associated services will fail as the circuit is no longer able to handle
+messages and the services have been stopped.
+
+For more information on the `disband` CLI command, see
+[`splinter-circuit-disband`](
+{% link docs/0.6/references/cli/splinter-circuit-disband.1.md %}).
+
+### Abandoning a circuit
+
+Abandoning a circuit differs from disbanding the circuit in that it is a single
+circuit member choosing to leave the circuit without agreement from the other
+members. A message is sent to all other circuit members that a circuit has been
+abandoned, but no further action is required by the other circuit members.
+From the abandoning node's perspective, once that message is sent, the circuit
+is deactivated. Specifically, this means the circuit is no longer able to route
+messages to other circuit members and the service associated with the abandoning
+node running on the abandoned circuit is stopped. The circuit state for the
+abandoning node is also updated to reflect the `Abandoned` status.
+
+#### Consequences
+
+Take care to note if a circuit has been abandoned by another node, as this
+means the abandoning node is unreachable via the abandoned circuit. This breaks
+the functionality of a circuit for other members as Splinter currently uses the
+Two-Phase Commit consensus algorithm which requires all nodes to verify their
+updated state. As such, any transactions will fail to commit for any of the
+other circuit members as the abandoning node is not available to reach
+consensus for a transaction.
+
+For more information on the `abandon` CLI command, see
+[`splinter-circuit-abandon`](
+{% link docs/0.6/references/cli/splinter-circuit-abandon.1.md %}).
+
+For information about configuring a Splinter node, see
+[Configuring Splinter Nodes](
+{% link docs/0.6/tutorials/configuring_splinter_nodes.md %}). In this guide,
+you'll be taking on the role of two different admins for two nodes (on the same
+system), so that you can create and test removing a circuit. For the first part
+of the guide, you will act as the first "alpha" node administrator to
+propose disbanding a circuit. Then, you will act as the second "beta" node
+administrator to accept the disband proposal. For the second part of this
+guide, you'll act as the first "alpha" node administrator to abandon a
+circuit. Next, you'll act as the second "beta" node administrator to view the
+"ABANDONED CIRCUIT" message from the "alpha" node. Finally, you will purge the
+inactive circuit.
+
+## Prerequisites
+
+* A splinter network with at least two nodes running and at least one circuit
+  created. The guide will refer to the two nodes as "alpha" and "beta"
+  respectively. The guide will refer to the already created circuit as
+  "01234-ABCDE". For more information on configuring a Splinter network, see
+  [Configuring Splinter Nodes]({%link
+  docs/0.6/tutorials/configuring_splinter_nodes.md %})
+
+* The splinter command line tool installed with
```

Comment:
This should work with stable CLI now right?

Reply:
Updated to remove this point